roles/cdi/defaults/main.yml

```diff
@@ -2,7 +2,7 @@ cdi_namespace: "{{ cdi_image_namespace | default('kube-system') }}"
 cdi_storageclass: "{{ cdi_kubevirt_storageclass | default('kubevirt') }}"
 apb_action: "provision"
 docker_prefix: "{{ cdi_repo_tag | default('kubevirt') }}"
-docker_tag: "{{ cdi_release_tag | default('v1.2.0') }}"
+cdi_release_tag: "v1.2.0"
```

Comment:
I believe we should use cdi 1.3

Reply:
correct. @shiywang already has a PR for it